test/expected/drop_chunks.out

```diff
@@ -122,13 +122,210 @@ CREATE VIEW dependent_view AS SELECT * FROM _timescaledb_internal._hyper_1_1_chu
 SELECT drop_chunks(2);
 ERROR:  cannot drop table _timescaledb_internal._hyper_1_1_chunk because other objects depend on it
 \set ON_ERROR_STOP 1
+-- Drop one chunk "manually" and verify that dimension slices and constraints are cleaned up
```

Comment:
Can we add a test where dimension_slices are shared and shouldn't be deleted by a chunk drop?